index.js

```diff
@@ -16,6 +16,14 @@ function promisifyOwnProperties(obj) {
 }
 
 module.exports = obj => {
+  if (!obj) {
+    return obj;
+  }
+
+  if (typeof obj !== 'function' && (typeof obj !== 'object' || Array.isArray(obj))) {
```

Comment:
I'll copy what I wrote there, I think it should be kept

Hmm, passing an array in makes a weird object.

I kept the test with an array just to see what happened. Is this wanted?

```
    Expected value to be (using ===):
      [1, true, "string", null, undefined, Symbol(some symbol)]
    Received:
      {"0": 1, "1": true, "2": "string", "3": null, "4": undefined, "5": Symbol(some symbol)}
```

An alternative is to loop through each element if it's an array and promisify functions there?